lib/rules/indentation/html.test.js

```diff
@@ -679,3 +679,141 @@ testRule({
 		},
 	],
 })
+testRule({
+	plugins,
+	ruleName,
+	config: [
+		2,
+		{
+			baseIndentLevel: 1,
+		},
+	],
+	customSyntax: `postcss-html`,
+	fix: true,
+
+	accept: [
+		{
+			code: stripIndent(`
```

Comment:
I don't think this should be an accepted test case.

[The current implementation](https://stylelint.io/demo/#N4Igxg9gJgpiBcIA8ALALgWwDYD4A6AdgARGowCGU+xJpAzmgJ5YzW23lHCFQCWdAByzlG8IgCMsEMAGsA3IXYkAvotpIA9A2as1mlBSqF9mXCAA0IAGa8WAOXIY4iGAA9HQmADowdOhfAIAhsAcwQQbho8cABXBggMAGVGAjRyV2ixaIEIBl86AFp0bGjzNWysGJDeAjpMogBtaIABbSx+NF4wLSYWdtSCoSqa6IBdMqiQACcYljqELjUSFraOro0a2FTyTqD6hoAmcy4iaPFyOhgASQIttAAZGAA3GCx6gEYiZVG1VQJlAKQYK8EIAMQgUwwO3CACs6EEArABP5EJFliA2q8amh6tFhGgYAxSuUQK1emtupj+mhBpVqgRcSB8YScRYSTk8n4iqZGcyiSBCADlEA) calculates the base indentation incorrectly, but works as expected for nested properties - in your implementation `display:block` is obviously placed in the wrong place